src/Generating/FinishGen.cpp

```diff
@@ -1763,7 +1782,16 @@ void cFinishGenOreNests::GenerateOre(
 		nestRnd /= cChunkDef::Width;
 		int BaseY = nestRnd % a_MaxHeight;
 		nestRnd /= a_MaxHeight;
-		int NestSize = a_NestSize + (nestRnd % (a_NestSize / 4));  // The actual nest size may be up to 1 / 4 larger
```

Comment:
Would std::min(4, a_NestSize) work

Reply:
calculating  x mod y with 0 < y < 1 results in realy big negative values so i think
```min(a_NestSize, abs(a_NestSize + (nestRnd % (a_NestSize / 4)))```
would do the thing here

Reply:
I wonder if the issue is that a_NestSize / 4 does an integer division, so if it's less than 4 it rounds to zero, which blows up the modulo with a divide by zero

Reply:
so i actually rounds to zero and then crashes with a division by zero

Reply:
you solution works but creates veines with at least one more as the defined size